big.in.sample

```diff
@@ -2,41 +2,41 @@
 ) Lines beginning with `)' are ignored.
 )---------------------------------------------------------------------
  style (Cartesian, Asteroidal, Cometary) = Cartesian
- epoch (in days) = 2451000.5
-)---------------------------------------------------------------------
- MERCURY    m=1.66013679527193009E-07 r=20.D0 d=5.43
- -3.83966017419175965E-01 -1.76865300855700736E-01  2.07959213998758705E-02
-  5.96286238644834141E-03 -2.43281292146216750E-02 -2.53463209848734695E-03
-  0. 0. 0.
- VENUS      m=2.44783833966454430E-06 r=20.d0 d=5.24
-  6.33469157915745540E-01  3.49855234102151691E-01 -3.17853172088953667E-02
- -9.84258038001823571E-03  1.76183746921837227E-02  8.08822351013463794E-04
-  0. 0. 0.
- EARTHMOO   m=3.04043264264672381E-06 r=20.d0 d=5.52
-  2.42093942183383037E-01 -9.87467766698604366E-01 -4.54276292555233496E-06
-  1.64294055023289365E-02  4.03200725816140870E-03  1.13609607260006795E-08
-  0. 0. 0.
- MARS       m=3.22715144505386530E-07 r=20.d0 d=3.94
-  2.51831018120174499E-01  1.52598983115984788E+00  2.57781137811807781E-02
- -1.32744166042475433E-02  3.46582959610421387E-03  3.98930013246952611E-04
-  0. 0. 0.
- JUPITER    m=9.54791938424326609E-04 r=3.d0 d=1.33
-  4.84143144246472090E+00 -1.16032004402742839E+00 -1.03622044471123109E-01
-  1.66007664274403694E-03  7.69901118419740425E-03 -6.90460016972063023E-05
-  0. 0. 0.
- SATURN     m=2.85885980666130812E-04 r=3.d0 d=0.70
-  8.34336671824457987E+00  4.12479856412430479E+00 -4.03523417114321381E-01
- -2.76742510726862411E-03  4.99852801234917238E-03  2.30417297573763929E-05
-  0. 0. 0.
- URANUS     m=4.36624404335156298E-05 r=3.d0 d=1.30
-  1.28943695621391310E+01 -1.51111514016986312E+01 -2.23307578892655734E-01
-  2.96460137564761618E-03  2.37847173959480950E-03 -2.96589568540237556E-05
-  0. 0. 0.
- NEPTUNE    m=5.15138902046611451E-05 r=3.d0 d=1.76
-  1.53796971148509165E+01 -2.59193146099879641E+01  1.79258772950371181E-01
-  2.68067772490389322E-03  1.62824170038242295E-03 -9.51592254519715870E-05
-  0. 0. 0.
- PLUTO      m=7.39644970414201173E-09 r=3.d0 d=1.1
- -1.15095623952731607E+01 -2.70779438829451422E+01  6.22871533567077229E+00
-  2.97220056963797431E-03 -1.69820233395912967E-03 -6.76798264809371094E-04
-  0. 0. 0.
+ epoch (in days) = 2457724.5D0
+)----------------------------------------------------d or D is not matter--0d0 is possible too
```

Comment:
The initial data was received from
http://ssd.jpl.nasa.gov/
by using HORIZONS web-interface (it provides several positions on a user-given time interval, and I took one of them)